plugins/community/nipunn1313-mypy/v3.6.0/requirements.txt

```diff
@@ -0,0 +1,3 @@
+mypy-protobuf==3.5.0
```

Comment:
Same note as above on how to generate this file. If you don't upgrade this, it isn't actually running with 3.6.0.